client/me/purchases/manage-purchase/index.jsx

```diff
@@ -182,10 +182,10 @@ class ManagePurchase extends Component {
 	}
 
 	handleRenew = () => {
-		const { purchase, siteSlug, redirectTo } = this.props;
+		const { purchase, siteSlug, redirectTo, dispatch } = this.props;
 		const options = redirectTo ? { redirectTo } : undefined;
 
-		handleRenewNowClick( purchase, siteSlug, options );
+		dispatch( handleRenewNowClick( purchase, siteSlug, options ) );
```

Comment:
This looks off. We should add `handleRenewNowClick` to the `mapDispatchToProps` object as a second argument to the `connect()` call below, in order to automatically bind it to `dispatch()` instead, and then invoke it directly here.